src/index.html

```diff
@@ -22,6 +22,420 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          src="../src/images/logo.png"
+          alt="header logo"
```

Comment:
alt attribute should describe the image content
<img width="461" alt="image" src="https://github.com/user-attachments/assets/cac885c9-62b1-47fc-a400-4b81036245a3">